_posts/2020-11-05-markdown-tutorial.markdown

```diff
@@ -0,0 +1,133 @@
+  * Item 2b
+```
+Ordered
+1. Item 1
+1. Item 2
+1. Item 3
+   1. Item 3a
+   1. Item 3b
+
+Unordered
+* Item 1
+* Item 2
+  * Item 2a
+  * Item 2b
+
+##### Horizontal lines
+Horizontal lines (---) are used to denote visual separation between different sections of a text. The syntax in block quote is showing what the text looks like in Markdown.
+```
+---
+```
+---
+##### Code snippets
+Code snippets are visible to readers of an article by using the back-tick symbol around a code in a sentence. For example;
+
+```
+```r
+#read in the csv
+all_covid_data <- read.csv("path/to/my/file.csv")```
+```
+```r
+#read in the csv
+all_covid_data <- read.csv("path/to/my/file.csv")
+```
+##### Inline code
+Inline codes are written by using the back tick and/or less and greater than symbol. The syntax in block quote is showing what the text looks like in Markdown.
+```
+For an inline code you can use
+`<this>` in the text line.
+```
+For an inline code you can use
+`<this>` in the text line.
+
+## Implementations of Markdown
+Implementations of Markdown are available for a lot of programming languages and applications and also multiple platforms support this Markup language.  
+In as much as Markdown is a minimal Markup language that is read and edited with the text editors, there are specially designed editors that are beneficial in previewing the files and these are available for all major platforms. Examples are;
+1. **Atom**: A desktop application built with HTML, JavaScript, CSS, and Node.js integration. It also serves as an Integrated Development Environment (IDE).
+2. **Doxygen**: A source code documentation generator which supports Markdown with extra features.
+3. **R Studios**: An Integrated Development Environment (IDE) for R language.
+4. **Sublime Text**: A sophisticated text editor for code, markup and prose.
+5. **GitHub Flavored Markdown (GFM)**: uses its own version of the Markdown syntax that provides an additional set of useful features, many of which make it easier to work with content on GitHub.com. It is useful for rendering README files in GitHub.
+6. **Discount**: A C implementation.
+7. **PHP Markdown**: A library package that includes the PHP Markdown parser and its sibling PHP Markdown Extra with additional features.
+8. **Markdig**: A .NET library that follows the CommonMark specifications and includes a collection of extensions and the ability for the user to create their own.
+9. **MarkAPL**: A converter written in Dyalog APL language.
+10. **Showdown and Smartdown**: Markdown renderers in JavaScript.
+11. **hackmd.io**: An online Markdown editor that supports Markdown with extra features.
+12. **gomarkdown**: Markdown parser and HTML renderer in Go.
+
+## Reasons you should use Markdown.
+1. It is suitable for everything. From writing online blog posts, to creating README files for projects and more.
+2. It is platform independent because it is a plaintext file. Markdown can be created on any device running Operating System.
```

Comment:
You can probably elaborate on some of these points.

Reply:
Perhaps, on line 1 you could say that RMarkdown is an easy way to provide structure and style to any of these documents with ease. Or something along those lines.

For 2 I would just say this is important because you don't need an proprietary software (like microsoft word) for Rmarkdown files.